src/en/gogoanime/src/eu/kanade/tachiyomi/animeextension/en/gogoanime/GogoAnime.kt

```diff
@@ -202,8 +202,8 @@ class GogoAnime : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
         val domainPref = ListPreference(screen.context).apply {
             key = "preferred_domain"
             title = "Preferred domain (requires app restart)"
-            entries = arrayOf("gogoanime.tel", "gogoanime.ar")
-            entryValues = arrayOf("https://gogoanime.tel", "https://gogoanime.ar")
+            entries = arrayOf("gogoanime.tel", "gogoanime.ar", "gogoanime.cl", "gogoanime.llc")
+            entryValues = arrayOf("https://gogoanime.tel", "https://gogoanime.ar", "https://gogoanime.cl", "https://gogoanime.llc")
```

Comment:
Since .tel redirects to .llc and .ar redirects to .cl, wouldn't it be better to replace rather than append?

Reply:
should i remove .tel  and .ar

Reply:
and keep only .cl & .llc new domains?

Reply:
Yes, since .cl and .llc are currently the only active domains. Any other (legit) domain either redirects to any of the two, or is just dead.

Reply:
Done thanks